CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute to CTF Handbooks
+
+## Making Changes to the Handbooks
+
+### Structure of the Writerside Folder
+```
+./Writerside/
+├───cfg/
+│   ├───buildprofiles.xml
+├───images/
+│   ├───air.png
+│   ├───appenditemname.png
+│   ├───...
+├───topics/
+│   ├───Commonly-Used-Itemstrings-List.md
+│   ├───Making-a-Map-for-Capture-the-Flag.md
+│   └───...
+├───c.list
+├───ch.tree
+├───v.list
+└───writerside.cfg
+```
+The main folders you would need to know are the [`images`](./Writerside/images) and [`topics`](./Writerside/topics) sub-folders within the [`Writerside`](./Writerside) folder.
+- [`images`](./Writerside/images): This folder contains all the image media used in the handbook. When adding new images, place them in this folder.
+- [`topics`](./Writerside/topics): This folder is the home to all the Handbook markdown files. When adding new chapters [without using Writerside](#without-using-writerside), make new markdown files for them here.
```

Comment:
I don't think it's possible to add new chapters without Writerside, since afaik the .md file needs to be configured by Writerside and written into the conf files 🤔

Reply:
Ah, right, that's what I had wanted to test and forgot.

Reply:
So it seems that adding a Markdown Topic in Writerside adds the markdown file in ./Writerside/topics/ and modifies ./Writerside/ch.tree . One could add the toc-element in ch.tree manually. Do you think we should mention that as a workaround?

Reply:
Your choice :P

Reply:
Hmm, I guess we will. 👌

Reply:
Okay then